receiver/README.md

```diff
@@ -15,32 +15,70 @@ Supported receivers (sorted alphabetically):
 - [Zipkin Receiver](#zipkin)
 
 ## Configuring Receiver(s)
-TODO - Add what a fullname is and how that is referenced in other parts of the
-configuration. Describe the common receiver settings: endpoint, disabled and
-disable-backpressure.
 
-## <a name="opencensus"></a>OpenCensus Receiver
-**Traces and metrics are supported.**
+Receivers are configured via YAML under the top-level `receivers` tag. There
+must be at least one enabled receiver for this configuration to be considered
+valid.
 
-This receiver receives spans from [OpenCensus](https://opencensus.io/) instrumented applications and translates them into the internal format sent to processors and exporters in the pipeline.
+The following is a sample configuration for the `examplereceiver`.
+```yaml
+receivers:
+  # Receiver 1.
+  # <receiver type>:
+  examplereceiver:
+    # <setting one>: <value one>
+    endpoint: 1.2.3.4:8080
+    # ...
+  # Receiver 2.
+  # <receiver type>/<name>:
+  examplereceiver/settings:
+    # <setting one>: <value one>
+    disabled: false
+    # <setting two>: <value two>
+    endpoint: 127.0.0.1:9211
+```
 
-Its address can be configured in the YAML configuration file under section "receivers", subsection "opencensus" and field "address". The syntax of the field "address" is `[address|host]:<port-number>`.
+A receiver instance is referenced by its full name in other parts of the config,
+such as in pipelines. A full name consists of the receiver type, '/' and the
+name appended to the receiver type in the configuration. All receiver full names
+must be unique. For the example above:
+- Receiver 1 has full name `examplereceiver`.
+- Receiver 2 has full name `examplereceiver/settings`.
 
-For example:
+Most receiver expose a setting to disable it, by default receivers are enabled.
+At least one receiver has to be enabled for this part of the configuration to be
```

Comment:
nit:  simpler sentance, if possible with a link to a pipeline descrition (TODO?)
```suggestion
At least one receiver is enabled in a valid pipeline configuration.
```

Reply:
Changed it to: `At least one receiver must be enabled to be a valid configuration.` yay/nay?